tests/test_A000578.py

```diff
@@ -0,0 +1,4 @@
+from oeis import A000578
+
+def test_A000578():
+    assert A000578[:20] == [0, 1, 8, 27, 64, 125, 216, 343, 512, 729, 1000, 1331, 1728, 2197, 2744, 3375, 4096, 4913, 5832, 6859]
```

Comment:
There's no new line at the end of this test (and all your other tests).

Please end all lines with a newline character, even the last line: it's prettier when the files gets concatenated, for example.